m3u8/read_write_test.go

```diff
@@ -296,6 +296,23 @@ func TestReadWriteExtXIFrameStreamInf(t *testing.T) {
 	}
 }
 
+func TestReadWriteMultipleExtXDateRange(t *testing.T) {
+	is := is.New(t)
```

Comment:
I suggest to drop this explicit test and incorporate the playlist in the `TestReadWritePlaylists` test